superset/assets/javascripts/explore/components/ExploreViewContainer.jsx

```diff
@@ -101,13 +106,43 @@ class ExploreViewContainer extends React.Component {
     }
   }
 
+  addHistory(isReplace = false, title) {
+    const { payload } = createExplore(this.props.form_data);
+    const longUrl = getExploreLongUrl(this.props.form_data);
+    if (isReplace) {
+      history.replaceState(
+        payload,
+        document.title,
+        longUrl);
+    } else {
+      history.pushState(
+        payload,
+        document.title,
+        longUrl);
+    }
+
+    // it seems browsers don't support pushState title attribute
```

Comment:
some do right? otherwise it wouldn't be an input to `push/replaceState`? maybe caveat with "some browsers"?